src/yoga.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import fs from 'node:fs';
+import {createRequire} from 'node:module';
+import {dirname, join} from 'node:path';
+import initYoga from 'yoga-wasm-web';
+
+const Yoga = await initYoga(
+	fs.readFileSync(
+		join(
+			dirname(
+				createRequire(import.meta.url).resolve('yoga-wasm-web/package.json')
+			),
+			'dist/yoga.wasm'
+		)
+	)
+);
+
+export default Yoga;
+export * from 'yoga-wasm-web';
+export type {Node as YogaNode} from 'yoga-wasm-web';
```

Comment:
I feel like `yoga-wasm-web` should handle all this boilerplate: https://github.com/shuding/yoga-wasm-web/issues/23

Reply:
Cool I see the issue is now closed thanks to @jeetiss in https://github.com/shuding/yoga-wasm-web/pull/24

Once that's released I'll apply it to this PR.

Reply:
it is released 🙌🏻 
[and works](https://github.com/vercel/satori/pull/429/commits/9a651dc1afdcf9b945d7af500298ab1503a16aeb)

Reply:
This is a lot cleaner now thanks to `/auto`, thanks @sindresorhus for raising this and @jeetiss for contributing it!